README.md

```diff
@@ -22,3 +22,43 @@ Working Group (WG) facilitates discussions and work on cross-SIGs. For example,
 - [Twitter](https://twitter.com/KubeSphere)
 - [Mailing List](https://groups.google.com/forum/#!forum/kubesphere)
 - [Medium (Blog)](https://itnext.io/@kubesphere)
+
+## Repository list // WIP
+
+|      |      |
+| ---- | ---- |
+| Project | Description |
```

Comment:
This line should be in line 28. And line 28 should be here.

Reply:
As you said earlier, we hope to keep the README file clean, and we have discussed that QingCloud can assign one member who familiar with the projects list in ks org to organize the first version of the repo list, so I don’t want to summit a new commit for this issue.

Reply:
In my opinion, we still can merge this PR and push it forward. The most important of community is improvement instead of perfect.

Reply:
I will take over this PR if you're not interested in this.